flyway/scxa/migrations/V18__scxa-create-exp-design.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE exp_design
+(
+    accession VARCHAR(255)
+        CONSTRAINT exp_design_k
+            REFERENCES experiment (accession)
+            ON DELETE CASCADE,
+    marray VARCHAR(255),
```

Comment:
This is never going to be filled for SCXA (no arrays there). I would not bother with the column at all and handle its lack at the application (I'm sure there's even SQL query magic to add literal blanks to match the output of the bulk query).